README.md

```diff
@@ -63,23 +63,23 @@ To get teleirc working, you will need a server to run it on and a recent version
 
 ##### Which image do I choose?
 
-Fedora, Ubuntu and Alpine Linux images are provided.
+Official Node 6 Slim, Official Node 6 Alpine Linux, and Fedora and images are provided.
 
 Their sizes may be deal breakers (ordered ascending by size):
 
-| **Image**                                                                            | **Size** |
-|--------------------------------------------------------------------------------------|----------|
-| Alpine Linux(uses [mhart/alpine-node:6](https://hub.docker.com/r/mhart/alpine-node)) | 345 MB   |
-| Ubuntu                                                                               | 598.4 MB |
-| Fedora                                                                               | 1.13 GB  |
+| **Image**                                                                                                    | **Size** |
+|--------------------------------------------------------------------------------------------------------------|----------|
+| [Official Node 6 Alpine Linux](https://hub.docker.com/r/_/node/)(`node:6-alpine`) -- See `Dockerfile.alpine` | 66.9 MB  |
+| [Offical Node 6 Slim](https://hub.docker.com/r/_/node/) (`node:6-slim`) -- See `Dockerfile.slim`           | 256 MB   |
```

Comment:
Minor OCD comment, since this should render correctly in markdown as-is, but I'd like to see you add a few more spaces so the columns line up in plaintext.